docs/README.md

```diff
@@ -1,45 +1,45 @@
 
-Caikit-TGIS-Serving is a combined image that allows data scientists to perform Large Language Model (LLM) inference.
+Caikit-TGIS-Serving is a stack that allows data scientists to perform Large Language Model (LLM) inference.
 
 The Caikit-TGIS-Serving stack consists of these components:
-* **[Text Generation Inference Server (TGIS)](https://github.com/opendatahub-io/text-generation-inference)**: The serving backend that loads the models and provides the inference engine.
-* **[Caikit](https://github.com/opendatahub-io/caikit)**: A wrapper layer that handles the lifecycle of the TGIS process, provides the inference endpoints, and has modules to handle different model types.
-* **[Caikit-nlp](https://github.com/opendatahub-io/caikit-nlp)**: The Caikit module that handles Natural Language Processing (NLP)-style models.
-* **[KServe](https://github.com/opendatahub-io/kserve)**: A Kubernetes Custom Resource Definition that orchestrates model serving for all types of models. It includes serving runtimes that implement the loading of given types of model servers. KServe handles the lifecycle of the deployment object, storage access, and networking setup.
-* **[Service Mesh](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-architecture.html)** (istio): The service mesh networking layer that manages traffic flows and enforces access policies.
-* **[Serverless](https://docs.openshift.com/serverless/1.29/about/about-serverless.html)** (knative): A cloud-native development model that allows for serverless deployments of data models.
 
+- **[Caikit](https://github.com/opendatahub-io/caikit)**: Caikit is an AI toolkit that enables users to manage models through a set of developer friendly APIs.
+- **[Caikit-nlp](https://github.com/opendatahub-io/caikit-nlp)**: Caikit module that handles Natural Language Processing (NLP) models and tasks.
+- **[Text Generation Inference Server (TGIS)](https://github.com/opendatahub-io/text-generation-inference)**: Runtime that loads the models and provides the inference engine.
+- **[KServe](https://github.com/opendatahub-io/kserve)**: A Kubernetes Custom Resource Definition that orchestrates model serving for all types of models. It includes serving runtimes that implement the loading of given types of model servers. KServe handles the lifecycle of the deployment object, storage access, and networking setup.
+- **[Service Mesh](https://docs.openshift.com/container-platform/4.13/service_mesh/v2x/ossm-architecture.html)** (istio): The service mesh networking layer that manages traffic flows and enforces access policies.
+- **[Serverless](https://docs.openshift.com/serverless/1.29/about/about-serverless.html)** (knative): A cloud-native development model that allows for serverless deployments of data models.
 
 ## Architecture of the stack
 
 ![KServe+Knative+Istio+Caikit_TGIS Diagram](https://github.com/opendatahub-io/caikit-tgis-serving/assets/8479010/7009b95d-0f6f-4f18-b0e6-355f360a5ad1)
```

Comment:
@heyselbi I see you're the last person to have updated this diagram. Do you think we should update it once again to reflect the new architecture?